apic.go

```diff
@@ -70,39 +70,39 @@ func yaml_string_read_handler(parser *yamlParser, buffer []byte) (n int, err err
 }
 
 // Reader read handler.
-func yaml_reader_read_handler(parser *yamlParser, buffer []byte) (n int, err error) {
+func yamlReaderReadHandler(parser *yamlParser, buffer []byte) (n int, err error) {
```

Comment:
Same as

https://github.com/yaml/go-yaml/pull/95#discussion_r2311923108

It could become


```suggestion
func (parser *yamlParser) readReader(buffer []byte) (n int, err error) {
```

Reply:
see https://github.com/yaml/go-yaml/pull/95#discussion_r2311993268